src/unix/linux_like/linux/mod.rs

```diff
@@ -2101,15 +2102,31 @@ pub const NFNL_SUBSYS_CTNETLINK_TIMEOUT: ::c_int = 8;
 pub const NFNL_SUBSYS_CTHELPER: ::c_int = 9;
 pub const NFNL_SUBSYS_NFTABLES: ::c_int = 10;
 pub const NFNL_SUBSYS_NFT_COMPAT: ::c_int = 11;
-pub const NFNL_SUBSYS_COUNT: ::c_int = 12;
+cfg_if! {
+    if #[cfg(target_env = "musl")] {
+        // FIXME: CI kernel headers lag bit behind;
+        // NFNL_SUBSYS_HOOK was added with v5.14.
+        pub const NFNL_SUBSYS_COUNT: ::c_int = 12;
+    } else {
+        pub const NFNL_SUBSYS_HOOK: ::c_int = 12;
+        pub const NFNL_SUBSYS_COUNT: ::c_int = 13;
+    }
+}
```

Comment:
The exception in build.rs should apply to both glibc and musl, so you shouldn't need to add this cfg_if.

Reply:
True. I could swear I submitted that in an earlier revision and it resulted in a CI failure,
prompting me to add that ``cfg_if``. Anyways, CI seems to be happy now without it.